Jenkinsfile

```diff
@@ -94,6 +94,11 @@ pipeline {
                             sh '''(cd /var/lib/mock/epel-7-x86_64/result/ &&
                                    cp -r . $OLDPWD/artifacts/centos7/)
                                   createrepo artifacts/centos7/'''
+                            publishToRepository product: 'slurm',
+                                                format: 'yum',
+                                                maturity: 'stable',
+                                                tech: 'el7',
```

Comment:
Seems an odd name for what is effectively `distro`.

Reply:
That naming convention was based https://jfrog.com/whitepaper/best-practices-structuring-naming-artifactory-repositories/